app/controllers/reservations_controller.rb

```diff
@@ -80,7 +80,7 @@ def success
 
     ReservationMailer.with(reservation: reservation).reservation_created_email.deliver_later
 
-    ActionCable.server.broadcast("notification_#{reservation.property.owner.id}", {action: "increase"})
+    ActionCable.server.broadcast("notification_#{property.owner.id}", {action: "increase", message: "Property at #{property.city} has been reserved", property_id: property.id})
```

Comment:
Hình như là noticed nó bắn broadcase theo từng type hay sao mà. Em check lại thử xem

Reply:
Dạ hiện em mới set noticed theo kiểu lưu về database chứ chưa bỏ qua ActionCable, tại ActionCable viết trước nên nếu xài noticed em sợ bị conflict, em sẽ tìm hiểu rồi refactor lại sau anh

Reply:
Tại anh nghĩ cái này thì dùng bằng noticed cũng được. Nên mình quy về 1 kiểu thôi cho dễ
```suggestion
    ActionCable.server.broadcast("notification_#{property.owner.id}", {action: "increase", message: "Property at #{property.city} has been reserved", property_id: property.id})
```